schemas/telemetry/crash/crash.4.schema.json

```diff
@@ -1296,18 +1300,34 @@
               "description": "<time>, Seconds since the Epoch (see also: `payload.crashTime`)",
               "type": "string"
             },
+            "DOMFissionEnabled": {
+              "description": "Set to 1 when DOM fission is enabled, and subframes are potentially loaded in a separate process.",
+              "type": "boolean"
```

Comment:
I have reason to believe this is not the correct type, and that all types within the metadata field should be a string. Let me put together a query to confirm this.